int32 target_id
int32 center_x
int32 center_y
int32 width
int32 height
---
bool success
string message